changelogs/fragments/7726-incus-plugin.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - "incus - add a new connection plugin for Incus"
```

Comment:
Please remove this file. New modules and plugins are announced by their `version_added` value and `short_description`, they don't need an explicit changelog fragment.